### ISSUE TITLE
Amplify Chrome tab crashes every now and then

### ISSUE THREAD

**Issue by janvitos** [nginxinc/nginx-amplify-agent#106]
Hi,

Since a few days, my Chrome tab that has the Amplify dashboard open keeps on crashing every now and then. I tried restarting Chrome and even my computer, but the problem persists.

I'm currently running Chrome Version 66.0.3359.139 (Official Build) (64-bit), Amplify agent version 1.2.0 on Centos 7 with NGINX 1.13.12.

At the time of writing this ticket, I've opened the dashboard in Firefox and Chrome Canary and I will monitor to see if it crashes there as well.

Is there anything else I could do to troubleshoot the issue?

Thank you.

**Comment by janvitos** [comment]
After doing some tests, I noticed Amplify also crashed a tab in Google Chrome Canary Version 68.0.3416.0 (Build officiel) canary (64 bits).

Firefox did not crash up to now.

**Comment by akamoroz** [comment]
Hi @janvitos,
Are you still having the same issue? If yes – pls contact to me with Intercom (chat button on the right bottom corner).

**Comment by janvitos** [comment]
Yes, the issue occurs 5-10 times every day (could be more if I refreshed the page each time it crashes).

I will contact you with the chat, thank you.
